.python-version

```diff
@@ -0,0 +1 @@
+3.13
```

Comment:
isn't

```
[project]
requires-python = ">= 3.13"
```

enough in main pyproject.yml?

Reply:
Probably. `uv init` generated it, but it doesn't seem necessary. I'll .gitignore it.